readme.md

```diff
@@ -59,6 +59,15 @@ the CLI documentation to find the pass-through arguments a given interaction req
 * A familiarity with Python is assumed.
 
 ## Setup
+Setup-Prerequisites
+1. Python 2.7
+2. pyyaml library
```

Comment:
Let's recommend using requirements.txt instead of specifying the individual packages.